recipes/whisper-cpp/all/conanfile.py

```diff
@@ -0,0 +1,159 @@
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+        if is_apple_os(self):
+            if not self.options.with_coreml:
+                self.options.rm_safe("coreml_allow_fallback")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+    def requirements(self):
+        if not is_apple_os(self):
+            if self.options.with_blas:
+                self.requires("openblas/0.3.20")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        deps = CMakeDeps(self)
+        deps.generate()
+
+        tc = CMakeToolchain(self)
+        tc.variables["WHISPER_BUILD_TESTS"] = False
+        tc.variables["WHISPER_BUILD_EXAMPLES"] = False
+
+        if self.options.shared:
+            tc.variables["BUILD_SHARED_LIBS"] = True
+        if self.options.sanitize_thread:
+            tc.variables["WHISPER_SANITIZE_THREAD"] = True
+        if self.options.sanitize_address:
+            tc.variables["WHISPER_SANITIZE_ADDRESS"] = True
+        if self.options.sanitize_undefined:
+            tc.variables["WHISPER_SANITIZE_UNDEFINED"] = True
+        if self.options.no_avx:
+            tc.variables["WHISPER_NO_AVX"] = True
+        if self.options.no_avx2:
+            tc.variables["WHISPER_NO_AVX2"] = True
+        if self.options.no_fma:
+            tc.variables["WHISPER_NO_FMA"] = True
+        if self.options.no_f16c:
+            tc.variables["WHISPER_NO_F16C"] = True
+
+        if is_apple_os(self):
+            if self.options.no_accelerate:
+                tc.variables["WHISPER_NO_ACCELERATE"] = True
+            if self.options.with_coreml:
+                tc.variables["WHISPER_COREML"] = True
+                if self.options.coreml_allow_fallback:
+                    tc.variables["WHISPER_COREML_ALLOW_FALLBACK"] = True
+        else:
+            if self.options.with_blas:
+                if Version(self.version) >= "1.4.2":
+                    tc.variables["WHISPER_OPENBLAS"] = True
+                else:
+                    tc.variables["WHISPER_SUPPORT_OPENBLAS"] = True
+
+        tc.generate()
+
+    def _patch_sources(self):
+        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
+                        "target_include_directories(${TARGET} PUBLIC",
+                        "target_include_directories(${TARGET} PUBLIC ${CMAKE_INCLUDE_PATH}")
```

Comment:
Is this patch really necessary?

Reply:
Yes for some reason the way it pulls in openblas (using FindBLAS) doesn't pull in the include directories so I need to manually add that bit to get it to actually find the openblas headers

Reply:
Oh, I see. Could we try one thing? Let's add this param to the requirement:

```python
self.requires("openblas/0.3.20", transitive_headers=True)
```

And remove that patch part. Is it working now?

Reply:
No that doesn't work either. It's still failing to build